public/css/common.less

```diff
@@ -197,7 +197,7 @@ div.show-more {
   margin-left: 1em / 1.333em; // 1em / h2 font size
 }
 
-.object-detail .plugin-output {
+.object-detail .check-command-output .plugin-output {
```

Comment:
Instead of adding a class to the collapsible container, please use `:not(.caption) > .plugin-output`